src/graphql/network.rs

```diff
@@ -592,9 +592,13 @@ macro_rules! from_key_value {
                     orig_port: val.orig_port,
                     resp_port: val.resp_port,
                     proto: val.proto,
-                    last_time: val.last_time,
                     $(
-                        $fields: val.$fields,
+                        $str_num_field: {
+                            StringNumber(val.$str_num_field)
+                        },
+                    )*
+                    $(
+                        $plain_field: val.$plain_field,
```

Comment:
Why don't we switch the order of these two, to match the order of the parameters of this macro definition?